lib/ui/src/components/sidebar/Heading.stories.tsx

```diff
@@ -168,11 +168,11 @@ export const CustomBrandImageTall: Story = () => {
         brand: {
           title: 'My Title',
           url: 'https://example.com',
-          image: 'https://via.placeholder.com/100x150',
+          image: 'https://via.placeholder.com/200x350',
         },
       }}
     >
-      <Heading menu={menuItems} />
```

Comment:
Can you explain this @tobiasljungstrom ?

This seems odd to add in this PR?

Reply:
Ah yes, I remember what it was now - the tall brand image triggers the bug, and if the heading doesn't have a skipLinkHref the skip link isn't focusable. I guess it isn't strictly necessary since it is not focused automatically in this story, but it is required if you want to verify the behaviour manually.